src/controllers/at-sce.controller.js

```diff
@@ -20,17 +24,16 @@ atSCEController.renderSigninForm = async (req, res) => {
   res.render("signin");
 };
 
-atSCEController.signin = async (req, res) => {
-  console.log("--> atSCEController.signin");
-
-  // Redirect
-  req.flash("success_msg", "User signin Successfully");
-  res.redirect("/calculator");
-};
+// AT-SCE - Signin 
+atSCEController.signin = passport.authenticate("local", {
+    successRedirect: "/calculator",
+    failureRedirect: "/signin",
+    failureFlash: true
+});
 
 // AT-SCE - Logout
 atSCEController.signout = async (req, res) => {
-  console.log("--> atSCEController.signout");
+  req.logout();
```

Comment:
can u keep this line  console.log("--> atSCEController.signout");

Reply:
sorry i made click, for mistake, in resolve but is not resolve yet